app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/sourceDetails/sourceDetails.jsx

```diff
@@ -46,18 +48,33 @@ export const SourceDetails = ({ info }) => {
     );
   };
 
+  const renderIssueActionBlock = () => {
+    return (
+      <div className={cx('issue-action-block')}>
+        <span className={cx('icon')}>{Parser(BugIcon)}</span>
+        <p>
+          {(issueActionType === POST_ISSUE && formatMessage(messages.postIssueNote)) ||
+            (issueActionType === LINK_ISSUE && formatMessage(messages.linkIssueNote)) ||
+            (issueActionType === UNLINK_ISSUE && formatMessage(messages.unlinkIssueNote))}
```

Comment:
It's better to simplify this condition, something like this
`{messages[`${issueActionType}Note`] ? formatMessage(messages[`${issueActionType}Note`]) : ''}`